website/views/pages/docs/basic-documentation.ejs

```diff
@@ -140,9 +140,14 @@
       </div>
   
       <h1 purpose="page-title" class="d-flex d-lg-none py-4 m-0">{{thisPage.title}}</h1>
+
+      <div class="edit-button-container">
+        <div class="edit-button">
+          <a class="edit-link" :href="'https://github.com/fleetdm/fleet/tree/main/docs/'+thisPage.sectionRelativeRepoPath"><i class="edit-pencil fa fa-pencil"></i>Edit page</a>
```

Comment:
same as above

Reply:
Also I'd advise leaving out any classes/purposes added for hypothetical future use (`edit-link`, `edit-pencil`).  They tend to get forgotten about.  (if these are here for a reason, then please ignore this feedback